### ISSUE TITLE
VTK

### ISSUE THREAD

**Issue by pelangho14** [ubccr/software-layer#363]
I previously attempted to install ITKSNAP using EasyBuild but was informed that there are no EasyBuild recipes available for this software. Therefore, I kindly request your assistance in installing ITKSNAP and its dependencies-VTK and ITK. Thank you.

**Comment by dsajdak** [comment]
@pelangho14 Thanks for submitting your build request.  As I mentioned in your help ticket we won't be able to install ITKSNAP.  However, we will attempt to to install the two dependencies.  We share the load of installing software among all staff so we ask that build requests list out only one software package per issue.  I'm going to modify this one to request the installation of VTK.  Please submit a new one for the installation of ITK so that you're notified when they're both installed.  Thank you!

**Comment by dsajdak** [comment]
vtk is available in `ccrsoft/2023.01`

```
$ module spider vtk

------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  vtk: vtk/9.1.0
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      The Visualization Toolkit (VTK) is an open-source, freely available software system for 3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several interpreted interface layers including
      Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques such as: implicit modeling, polygon reduction, mesh
      smoothing, cutting, contouring, and Delaunay triangulation.

    Properties:
      Visualisation software

    You will need to load all module(s) on any one of the lines below before the "vtk/9.1.0" module is available to load.

      gcc/11.2.0  openmpi/4.1.1

    Help:

      Description
      ===========
      The Visualization Toolkit (VTK) is an open-source, freely available software system for
       3D computer graphics, image processing and visualization. VTK consists of a C++ class library and several
       interpreted interface layers including Tcl/Tk, Java, and Python. VTK supports a wide variety of visualization
       algorithms including: scalar, vector, tensor, texture, and volumetric methods; and advanced modeling techniques
       such as: implicit modeling, polygon reduction, mesh smoothing, cutting, contouring, and Delaunay triangulation.


      More information
      ================
       - Homepage: https://www.vtk.org
```
